services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -529,9 +529,7 @@ async def add_project_states_for_user(
                 if prj_node is None:
                     continue
                 node_state_dict = json.loads(
-                    node_state.json(
-                        by_alias=True, exclude_unset=True, exclude={"current_status"}
-                    )
+                    node_state.json(by_alias=True, exclude_unset=True)
```

Comment:
if you exclude unset, ``node_state`` will not return fields with defaults? What about ``exclude_null``. I spoke to @odeimaiz and he told me that he could manage that in the front-end

Reply:
as discussed. exclude_none removes fields that were SET as None. which is not something we want.